win32ss/user/ntuser/focus.c

```diff
@@ -1490,7 +1490,7 @@ IntAllowSetForegroundWindow(DWORD dwProcessId)
    ppi = NULL;
    if (dwProcessId != ASFW_ANY)
    {
-      if (!NT_SUCCESS(PsLookupProcessByProcessId((HANDLE)dwProcessId, &Process)))
+      if (!NT_SUCCESS(PsLookupProcessByProcessId(UlongToHandle(dwProcessId), &Process)))
```

Comment:
Shouldn't we get a handle in the first place ?

Reply:
In win32 land process ids are usually DWORD.